gunicorn/datadog_checks/gunicorn/data/conf.yaml.example

```diff
@@ -3,6 +3,13 @@
 
 init_config:
 
+  ## @param gunicorn - string - optional
+  ## command or path to gunicorn (e.g. `/usr/local/bin/gunicorn` or `docker exec container gunicorn`)
+  ## can be overwritten on an instance
+  ## default: gunicorn
+  #
+  # gunicorn: gunicorn
```

Comment:
Note to reviewers: This is a similar implementation to https://github.com/DataDog/integrations-core/blob/master/cassandra_nodetool/datadog_checks/cassandra_nodetool/data/conf.yaml.example#L12